content/6_Advanced/Extend_Euclid.mdx

```diff
@@ -95,6 +121,42 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+def extend_euclid(a: int, b: int) -> list[int]:
+    """
+    Returns a list of Bézout coefficients and the
+    GCD of two positive, non-zero numeric values.
+    """
+
+    assert int(a) > 0 and int(b) > 0, \
+        "Arguments must be positive, non-zero numeric values."
+
+    x_arr: list[int] = [1, 0, int(a)]
+    y_arr: list[int] = [0, 1, int(b)]
+    q: int = -1
+
+    def swap(x, y):
+        tmp = x
+        x = y
+        y = tmp
+        return x, y
+
+    # We know that:
+    #   (1*a) + (0*b) == a
+    #   (0*a) + (1*b) == b
```

Comment:
multiline this

Reply:
I don't understand. Can you please clarify "multiline this"?

Reply:
use:
```py
"""
We know that:
etcetc
etcetc
"""
```

Reply:
Ah, gotcha.  Thanks for the visual!